meson.build

```diff
@@ -0,0 +1,132 @@
+project('transmission-remote-gtk', 'c',
+        license: 'GPL2+',
+        version: '1.4.2',
```

Comment:
If you set a minimum `meson_version: '>= ....'` then meson will make sure that you have a recent enough version and fail for users whose version is too old. Meson also warns you when you use deprecated functionality.

Based on a quick check, the most recent feature you used is: ` * 0.49.0: {'/ with string arguments'}`

Reply:
Oh, I see your updated comment that .allowed() requires a more recent version, which... meson doesn't issue a FeatureNew warning for. We should fix that in meson...